README.md

```diff
@@ -1 +1 @@
-docs/01-readme.md
\ No newline at end of file
+docs/01-readme.md
```

Comment:
please rebase out this change at all

Reply:
deleted

Reply:
i mean change rebase out the change to the symlink -- leave it without changes

Reply:
it still has no effective difference, and I can't figure out how to "rebase" it.